lib/players/human.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "../player.rb"
+
+class Players
+
+  class Human < Player
+
+    def move(board)
+      puts "Where would you like to place your move player #{self.token}? (a number between 1 and 9)"
+      puts ""
+      input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```